internal/controller/base_controller.go

```diff
@@ -8,7 +8,7 @@ import (
 	"time"
 
 	"github.com/go-logr/logr"
-	"github.com/ngrok/ngrok-api-go/v5"
+	"github.com/ngrok/ngrok-api-go/v6"
```

Comment:
This was because it wasn't catching my 404's on delete with the ngrok.IsNotFound error checker down below. Do we still have this on v5 just for passivity for some apis or something?

Reply:
#456 has been merged, we don't need to worry about this anymore.